modules/electromagnetics/test/include/functions/CosTheta.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "Function.h"
+
+/**
+ *  Function for cosine(theta) (where theta is in degrees) for use in slab
+ *  reflection benchmark.
+ */
```

Comment:
there's cos in parsed?

Reply:
I specifically made this test object long ago in order to super easily script and obtain results for the relevant slab reflection benchmark leveraging CLI args. No longer really necessary, so can be removed and replaced with something else that is parsed.